CarboPlanner/src/views/Loading.tsx

```diff
@@ -184,17 +175,20 @@ export default function Loading({route}) {
 
     //const api_url = process.env.EXPO_PUBLIC_API_URL;
```

Comment:
```suggestion
    const api_url = process.env.EXPO_PUBLIC_API_URL;
```

Please change it back to using the .env file